umh-core/pkg/service/redpanda_monitor/metrics_state.go

```diff
@@ -63,9 +65,21 @@ func NewRedpandaMetricsState() *RedpandaMetricsState {
 
 // UpdateFromMetrics updates the metrics state based on new metrics
 func (s *RedpandaMetricsState) UpdateFromMetrics(metrics Metrics, tick uint64) {
-	// Update component throughput
-	s.updateComponentThroughput(&s.Input, metrics.Throughput.BytesIn, tick)
-	s.updateComponentThroughput(&s.Output, metrics.Throughput.BytesOut, tick)
+	// Update component throughput in parallel
```

Comment:
How long does it take so that we parallelize it?

Reply:
Good point, here using a goroutine was not needed, as the difference was in ns.